cmd/copy.go

```diff
@@ -53,30 +66,32 @@ func Copy(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func isLocalDirToRemoteCopy(args []string) bool {
-	if strings.Contains(args[0], "sess:") {
+func shouldCopyLocalDirToRemote(localPath1, remotePath2 string) bool {
```

Comment:
I suggest not adding numbers to variable names.

Reply:
for context, this is to help clarify the local path should be the first argument and the remote path should be the second argument, i.e. given a list of args["sess:remote/path", "/local/path"] or the converse `args[ "/local/path", "sess:remote/path"]` it's still not clear from the variable naming alone, the first argument needs to be the local path and the second argument needs to be the remote path or vice versa.

so, im not sure what a good solution to this is other than to indicate localPath1 as 'the first argument that we _also_ expect to be the local path' and 'remotePath2' as 'the second argument that we _also_ expect to be the remotePath'.

any other ideas?

Reply:
edit: i've updated this to remove the second argument on the whole, seemed superfluous

Reply:
The variable names in a function signature should ideally be descriptive and not denote their positions. The position of a parameter in a function is determined by its place in the function definition itself, not by its name.

I suggest
`func shouldCopyLocalDirToRemote(localPath, remotePath string) bool {`

Reply:
i think in this case the intent of the positions was to also denote directionality, but yes this might be overthought, either way it's been removed now, thanks for commenting !

Reply:
I'd agree to keep numbers out of names. I don't think there's a rule but just looks off. 
[ Shorter names are better](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#variable-names). So in this case some thing `a` and `b` is idiomatic in go. The [standard library uses it all over the place](https://pkg.go.dev/sort#IntSlice.Less).

Reply:
So instead of `func shouldCopyLocalDirToRemote(remotePath)` we use `func shouldCopyLocalDirToRemote(from string)`

Ideally I would have

`func shouldCopyLocalDirToRemote(from Path)` where Path aliases a type declaration for filepaths that have been validated, but I think that's a bit much for our use cases.